http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/fault/tolerance/ClientRetryResource.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.ts.http.restclient.reactive.fault.tolerance;
+
+import java.util.concurrent.CompletionStage;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.ProcessingException;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+import org.eclipse.microprofile.faulttolerance.Retry;
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+@ApplicationScoped
+@RegisterRestClient(configKey = "client.endpoint")
+@Path("/client")
+public interface ClientRetryResource {
```

Comment:
You can call end this client either with `client` or `service` but I don't think they are called resources.